homeassistant/components/netatmo/sensor.py

```diff
@@ -112,84 +95,136 @@
 MODULE_TYPE_RAIN = "NAModule3"
 MODULE_TYPE_INDOOR = "NAModule4"
 
-
-NETATMO_DEVICE_TYPES = {
-    "WeatherStationData": "weather station",
-    "HomeCoachData": "home coach",
+BATTERY_VALUES = {
+    MODULE_TYPE_WIND: {"Full": 5590, "High": 5180, "Medium": 4770, "Low": 4360},
+    MODULE_TYPE_RAIN: {"Full": 5500, "High": 5000, "Medium": 4500, "Low": 4000},
+    MODULE_TYPE_INDOOR: {"Full": 5500, "High": 5280, "Medium": 4920, "Low": 4560},
+    MODULE_TYPE_OUTDOOR: {"Full": 5500, "High": 5000, "Medium": 4500, "Low": 4000},
 }
 
 PUBLIC = "public"
 
 
-async def async_setup_entry(
-    hass: HomeAssistant, entry: ConfigEntry, async_add_entities
-):
+async def async_setup_entry(hass, entry, async_add_entities):
     """Set up the Netatmo weather and homecoach platform."""
-    auth = hass.data[DOMAIN][entry.entry_id][AUTH]
     device_registry = await hass.helpers.device_registry.async_get_registry()
+    data_handler = hass.data[DOMAIN][entry.entry_id][DATA_HANDLER]
+    temp_data_classes = []
```

Comment:
It looks like `temp_data_classes` is redefined below in the nested function.

Reply:
Yes, knowingly. I can use a different name.

Reply:
I fixed that.